src/main/java/pl/nextleveldev/smart_route/route/api/RouteController.java

```diff
@@ -0,0 +1,32 @@
+package pl.nextleveldev.smart_route.route.api;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.http.ResponseEntity;
+import pl.nextleveldev.smart_route.route.RouteService;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/route")
+@RequiredArgsConstructor
+public class RouteController {
+
+    private final RouteService routeService;
+
+    @GetMapping("/search")
```

Comment:
You should avoid using verb in endpoints, my proposition is "/routes?from=Origin&to=Destination